crates/ra_ide_api/src/change.rs

```diff
@@ -134,7 +133,7 @@ impl LibraryData {
         root_id: SourceRootId,
         files: Vec<(FileId, RelativePathBuf, Arc<String>)>,
     ) -> LibraryData {
-        let symbol_index = SymbolIndex::for_files(files.par_iter().map(|(file_id, _, text)| {
+        let symbol_index = SymbolIndex::for_files(files.iter().map(|(file_id, _, text)| {
```

Comment:
I feel like we should just `#[cfg(feature = "wasm")]` this: rayon is pretty important for perf

Reply:
you mean `target_arch="wasm32"`?

Reply:
Hm, good question....

I feel like it's better to use Cargo feature, but that's just a gut feeling, so I am fine either way!